src/main/java/biomesoplenty/api/sound/BOPSounds.java

```diff
@@ -11,5 +11,7 @@
 
 public class BOPSounds
 {
-    public static SoundEvent music_disc_wanderer;
+	
+	public static SoundEvent music_disc_wanderer;
+	public static String MUSIC_WANDERER_NAME = "music_disc.wanderer";
```

Comment:
I would prefer to just duplicate this string in ModSounds and ModItems, as we don't use this style anywhere else.